scipy/linalg/tests/test_decomp_cholesky.py

```diff
@@ -8,17 +8,23 @@
 
 from scipy.linalg._testutils import assert_no_overwrite
 
+from scipy.conftest import (
+    array_api_compatible, skip_if_array_api, set_assert_allclose
+)
 
 class TestCholesky:
 
-    def test_simple(self):
-        a = [[8, 2, 3], [2, 9, 3], [3, 3, 6]]
+    @array_api_compatible
+    def test_simple(self, xp):
+        a = xp.asarray([[8, 2, 3], [2, 9, 3], [3, 3, 6]], dtype=xp.float64)
         c = cholesky(a)
-        assert_array_almost_equal(dot(transpose(c), c), a)
-        c = transpose(c)
-        a = dot(c, transpose(c))
-        assert_array_almost_equal(cholesky(a, lower=1), c)
+        _assert_allclose = set_assert_allclose(xp)
+        _assert_allclose(xp.matmul(c.T, c), a)
+        c = c.T
+        a = xp.matmul(c, c.T)
+        _assert_allclose(cholesky(a, lower=1), c)
```

Comment:
would it be more readable to have `lower=True` rather than `lower=1`?

Reply:
Yes, using True/False when a keyword is boolean is always better.